<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QSpinBox" name="spinBox_ip_3">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="8">
     <widget class="QChartView" name="graphicsView"/>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="spinBox_ip_2">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_freq">
      <property name="text">
       <string>Frequency</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QSpinBox" name="spinBox_freq">
      <property name="suffix">
       <string>ms</string>
      </property>
      <property name="minimum">
       <number>50</number>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
      <property name="singleStep">
       <number>100</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QSpinBox" name="spinBox_ip_4">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_ip">
      <property name="text">
       <string>IP address</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7" rowspan="2">
     <widget class="QLCDNumber" name="lcdNumber"/>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBox_ip_1">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item row="0" column="6" rowspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="pushButton_ping">
      <property name="text">
       <string>ping</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QCheckBox" name="checkBox_ping">
      <property name="text">
       <string>auto ping</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
